src/PayID/pay-id-client.ts

```diff
@@ -14,6 +9,13 @@ import PayIDError from './pay-id-error'
  * TODO(keefertaylor): Export this class in index.ts when it's ready for external consumption.
  */
 export default class PayIDClient {
+  /** An HTTP client. */
+  private readonly axiosInstance: AxiosInstance
+
+  public constructor() {
+    this.axiosInstance = axios.create({})
+  }
+
```

Comment:
Do we need an axios instance here? We could have this class only have static methods.

Reply:
I assumed there was an overhead in running `axios.create`. I also assumed that when I get a PayID spec that we might have some common headers which run across methods, etc. 

Is this an un-idiomatic use of axios?

Reply:
there's a global axios instance you can use if you want, which seems fine here. you can also set configs for it

Reply:
If there is common headers, common errors we are intercepting, or a common url then keep this but if we are just having one api just use the `axios.get` function and don't require a class initialization